### PR TITLE
Fixed example for creating ValidationError

### PR DESCRIPTION
**Pull request by BostjanPisler**

### CODE REVIEW

**Comment by slnode** [comment]
Can one of the admins verify this patch? To accept patch and trigger a build add comment ".*ok\W+to\W+test.*"

**Comment by slnode** [comment]
Can one of the admins verify this patch?

**Comment by slnode** [comment]
Can one of the admins verify this patch?

**Comment by slnode** [comment]
Can one of the admins verify this patch?

**Comment by Amir-61** [comment]
@slnode test please

File: lib/validations.js [hunk 1]
```diff
@@ -776,7 +776,7 @@ function ErrorCodes(messages) {
  *   }
  *
  *   // This line shows how to create a ValidationError
- *   err = new ValidationError(model);
+ *   var err = new MyModel.ValidationError(model);
```
Comment (Amir-61) [inline lib/validations.js]: I believe this should be `var err = new ValidationError(model);` for example see how it is used in other places as well: https://github.com/strongloop/loopback-datasource-juggler/blob/5bf18b07280e22d713e722b8e23f8aee14c6587e/lib/relation-definition.js#L2195

Comment (BostjanPisler) [inline lib/validations.js]: Using it like that in a remote method throws a `ValidationError is not defined` error.
Using proposed method works, the ideas is from here: https://github.com/strongloop/loopback/blob/master/lib/persisted-model.js#L438

Comment (Amir-61) [inline lib/validations.js]: Thanks for clarification!

**Pull request merged**